### PR TITLE
Provide e-mail and avatar inside basic auth.info

### PR DESCRIPTION
**Pull request by murbanski**

### CODE REVIEW

File: lib/omniauth/class_link/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module Omniauth
   module ClassLink
-    VERSION = "0.3.0"
+    VERSION = "0.3.1"
```
Comment (kamilbednarz) [inline lib/omniauth/class_link/version.rb:3]: Please do not change VERSION in pull requests. 
It's up to gem maintainer to bump the version.

File: lib/omniauth/strategies/class_link.rb [hunk 2]
```diff
@@ -4,7 +4,7 @@
 module OmniAuth
   module Strategies
     class ClassLink < OmniAuth::Strategies::OAuth2
-      option :name, :classlink
+      option :name, :class_link
```
Comment (kamilbednarz) [inline lib/omniauth/strategies/class_link.rb:7]: It's a breaking change. Please remove it.

**Review by kamilbednarz — CHANGES_REQUESTED** [review]

**Comment by murbanski** [comment]
Right, sorry about the burden.

**Comment by kamilbednarz** [comment]
Hey, I added the image field & published 0.3.1 version of the gem in Rubygems.

Email was added in a separate pull request. It's added in 0.3.1 too.

**Comment by murbanski** [comment]
❤️ so much!

**Pull request closed without merging**